### ISSUE TITLE
training error

### ISSUE THREAD

**Issue by jack155861** [lizhengwei1992/Semantic_Human_Matting#32]
hi

when I train this code and I get error
`
       
        for i, sample_batched in enumerate(trainloader):
            img, trimap_gt, alpha_gt = sample_batched['image'], sample_batched['trimap'], sample_batched['alpha']
            img, trimap_gt, alpha_gt = img.to(device), trimap_gt.to(device), alpha_gt.to(device)

            trimap_pre, alpha_pre = model(img)

            loss, L_alpha, L_composition, L_cross = loss_function(img,
                                                                  trimap_pre, 
                                                                  trimap_gt, 
                                                                  alpha_pre, 
                                                                  alpha_gt)

            optimizer.zero_grad()
            loss.backward()
            optimizer.step()

            loss_ += loss.item()
            L_alpha_ += L_alpha.item()
            L_composition_ += L_composition.item()
            L_cross_ += L_cross.item()

`

![image](https://user-images.githubusercontent.com/45505414/65828490-ffd78c00-e2cd-11e9-820a-4a1b2c08efdf.png)

how should i solve it ?

thanks

**Comment by jack155861** [comment]
oh i finished it thanks

**Comment by jack155861** [comment]
oh i finishes it  thanks
